docs/metrics.md

```diff
@@ -50,6 +50,14 @@ receivers:
 
 You can use the [OpenTelemetry operator](https://opentelemetry.io/docs/kubernetes/operator/) to setup the collector and instrument the workflow-controller.
 
+You can adjust temporality of the OpenTelemetry metrics configuration by changing values in the [Workflow Controller Config Map](workflow-controller-configmap.md).
```

Comment:
```suggestion
You can configure the temporality of OpenTelemetry metrics in the [Workflow Controller ConfigMap](workflow-controller-configmap.md).
```

- ConfigMap is one word (also UpperCamelCase, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#api-resource-names))
- simplify -- lot of redundant words here

Reply:
OTel link for temporality would be good here too

Reply:
Link added